src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Aggregator/FunctionResultLog.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host.Loggers
+{
+    internal class FunctionResultLog
```

Comment:
I remember looking at it and if I recall, it looked kinda ugly, but let me take another look since it's been a while.